### ISSUE TITLE
Error while indexing oxSectorIdentifierURI LDAP attribute

### ISSUE THREAD

**Issue by iromli** [GluuFederation/community-edition-setup#160]
During indexing LDAP attribute, I encountered an error as seen below:

```
The Backend Index could not be created because of the following reason:

*  Unwilling to Perform: The Directory Server is unwilling to add configuration entry
ds-cfg-attribute=oxSectorIdentifierURI,cn=Index,ds-cfg-backendid=userRoot,cn=Backends,cn=config
because one of the add listeners registered with the parent entry
cn=Index,ds-cfg-backend-id=userRoot,cn=Backends,cn=config rejected this change with the message: 
The Backend Index could not be decoded due to the following reason: 
The value "oxSectorIdentifierURI" is not a valid value for the "attribute" property, which must have the following syntax: OID
```

Digging the https://github.com/GluuFederation/community-edition-setup/blob/version_2.4.4/static/opendj/opendj_index.json#L45, my guess is `oxSectorIdentifierURI` is not recognized anymore, as `101-ox.ldif` doesn't specify `oxSectorIdentifierURI`.

After modified the https://github.com/GluuFederation/community-edition-setup/blob/version_2.4.4/static/opendj/opendj_index.json#L45 from `oxSectorIdentifierURI` into `oxSectorIdentifier`, i didn't see the error above.

Can someone check or confirm this issue?

**Comment by qbert2k** [comment]
oxSectorIdentifierURI was changed to oxSectorIdentifier to fix the issue oxAuth #186 Pairwise ID: Algorithm should use sector identifier not URI

**Comment by iromli** [comment]
Thanks for the confirmation @qbert2k.

One question though, do we need to change https://github.com/GluuFederation/community-edition-setup/blob/version_2.4.4/static/opendj/opendj_index.json#L45 or leave it as is?

**Comment by iromli** [comment]
I think it's fixed in 693461fbe63b63145563f5d7c0b7d6bbbc0147a5.
